### ISSUE TITLE
New folders are created with wrong permissions

### ISSUE THREAD

**Issue by mpryvkin** [trippo/ResponsiveFilemanager#427]
When `execute.php?action=create_folder` is called, function `create_folder()` creates a folder with permission `0755` instead of the one defined by `folderPermission` variable.

Workaround is to modify `execute.php` and add `chmod` as shown below:

````
      case 'create_folder':
         if ($create_folders)
         {

            $name = fix_filename($_POST['name'],$config);
            $path .= $name;
            $path_thumb .= $name;
            create_folder(fix_path($path,$config),fix_path($path_thumb,$config),$ftp,$config);
            chmod(fix_path($path,$config), $config['folderPermission']);
         }
         break;
````

**Comment by trippo** [comment]
The solution is to change in the function inutile.php at line 466

if ($path && ! file_exists($path))
		{
			$permission = 0755;
			if(isset($config['folderPermission']))){
				$permission = $config['folderPermission'];
			}
			mkdir($path, $permission, true);
		}

Thanks to report me this bug, I’ll include in next release

**Comment by mpryvkin** [comment]
This has been fixed in 9.13.1
